scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -610,7 +610,7 @@ def test_masked_stat_1d():
     np.testing.assert_array_equal(res6, res)
 
 
-@pytest.mark.parametrize(("axis"), range(-2, 2))
+@pytest.mark.parametrize(("axis"), range(-3, 3))
```

Comment:
This test (already merged) was supposed to work for this range, but there was a bug before. It's fixed now.